cleanlab/segmentation/summary.py

```diff
@@ -0,0 +1,332 @@
+# Copyright (C) 2017-2023  Cleanlab Inc.
+# This file is part of cleanlab.
+#
+# cleanlab is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# cleanlab is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with cleanlab.  If not, see <https://www.gnu.org/licenses/>.
+
+"""
+Methods to display images and their label issues in a semantic segmentation classification dataset, as well as summarize the types of issues identified.
+"""
+
+from typing import Any, Dict, List, Optional, Tuple
+
+import numpy as np
+import pandas as pd
+from tqdm import tqdm
+
+from cleanlab.internal.token_classification_utils import color_sentence, get_sentence
+
+
+def display_issues(
```

Comment:
Change `display_issues()` to `visualize()` an make it work for non issues as well

Reply:
I think I prefer display_issues over visualize but we can def talk about it - right now the other code really depends on this to just show erroneous so it might change things